nengo_viz/static/viz.css

```diff
@@ -1,3 +1,8 @@
 .graph {
-    background: #eeeeee;
-}
\ No newline at end of file
+    background: white;
+    border: 1px solid white;
```

Comment:
I'm thinking that if the backgrounds are changed to white, the border should be visible(non-white, perhaps a grey shade), this would give a better sense of where the components are.